www/_template/guides/connecting-tools.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: layouts/guide.njk
+title: The Snowpack Guide to connecting your favorite tools
+description: 'How do you use your favorite tools in Snowpack? This Guide will help you get started'
+tags: guides
+sidebarTitle: Connecting your favorite tools
+---
+
+One of the biggest questions we get in our discussion forums and Discord is how to connect tools to Snowpack. EsLint, PostCSS, SASS, and any other of the useful tools developers use to build JavaScript these days. In this Guide we'll go over how to use your favorite tools with Snowpack. Almost all work with just a few additions to a configuration file.
+
+## The three ways
+
+It's worth first going over the three ways and their pros and cons:
+
+- Snowpack plugins
+- Run scripts using Snowpack
+- Run scripts outside of Snowpack (in `package.json`)
+
+## Plugin
+
+Usually Snowpack plugins are the easiest way to use your favorite tools. Snowpack plugins are built to require no configuration, but if you do want to do configuration, they are built with that in mind too!
+
```

Comment:
"Snowpack has a [growing plugin ecosystem](/plugins)." Focus on this a bit more! Something like:

> The best way to connect a new tool to Snowpack is to search our [plugin catalog](/plugins) for a relevant plugin. Most likely, someone already created a plugin to help you integrate your favorite tool with ease.

> Adding an existing plugin to your Snowpack configuration involves two steps: ...